attestation-service/src/policy_engine/opa/mod.rs

```diff
@@ -10,19 +10,24 @@ use sha2::{Digest, Sha384};
 use std::collections::HashMap;
 use std::fs;
 use std::path::PathBuf;
+use std::sync::{Arc, Mutex};
 
 use super::{EvaluationResult, PolicyDigest, PolicyEngine, PolicyError};
 
-#[derive(Debug, Clone)]
+use crate::RvpsApi;
+
+#[derive(Clone)]
 pub struct OPA {
     policy_dir_path: PathBuf,
+    rvps: Arc<Mutex<dyn RvpsApi + Send + Sync>>,
```

Comment:
We can move the parameter to `evaluate` API than a member of `OPA` struct, which can help to avoid deliver the parameter in too many levels.